toto/runlib.py

```diff
@@ -100,6 +100,11 @@ def record_artifacts_as_dict(artifacts):
     return artifacts_dict
 
   for artifact in artifacts:
+
+    if not os.path.exists(artifact):
+      log.warning("path: {} does not exist, skipping..".format(artifact))
+      continue 
+
```

Comment:
Do you mean [`os.path.lexists()`](https://docs.python.org/2/library/os.path.html#os.path.lexists)?